### ISSUE TITLE
env('DB_CONNECTION')  is null when config have been cached

### ISSUE THREAD

**Issue by tanteng** [thedevdojo/voyager#289]
- Laravel Version: #.#.#
- Voyager Version: #.#.#
- PHP Version:
- Database Driver & Version:

### Description:
```
php artisan cache:clear
php artisan route:cache
php artisan config:cache
```
If run the command to cache config, you can see the exception 404 not found, then run php artisan route:list,you can't find the route for posts,users,categories and so on.

Exception:

```
NotFoundHttpException in compiled.php line 8912


    in compiled.php line 8912
    at RouteCollection->match(object(Request)) in compiled.php line 8236
    at Router->findRoute(object(Request)) in compiled.php line 8177
    at Router->dispatchToRoute(object(Request)) in compiled.php line 8173
    at Router->dispatch(object(Request)) in compiled.php line 2471
    at Closure$Illuminate\Foundation\Http\Kernel::dispatchToRouter(object(Request)) in Pipeline.php line 53
    at Closure$Illuminate\Routing\Pipeline::getInitialSlice(object(Request)) in compiled.php line 3226
    at CheckForMaintenanceMode->handle(object(Request), object(Pipeline::getInitialSlice;1945972947)) in compiled.php line 9900
    at Closure$Illuminate\Pipeline\Pipeline::getSlice(object(Request)) in Pipeline.php line 33
    at Closure$Illuminate\Routing\Pipeline::getSlice(object(Request)) in compiled.php line 9885
    at Pipeline->then(object(Kernel::dispatchToRouter;1270569954)) in compiled.php line 2415
    at Kernel->sendRequestThroughRouter(object(Request)) in compiled.php line 2399
    at Kernel->handle(object(Request)) in index.php line 54
```
tcg/voyager/routes/web.php line 20:
```
if (env('DB_CONNECTION') !== null && Schema::hasTable('data_types')):
```
env('DB_CONNECTION')  is null when config have been cached.

**Comment by marktopper** [comment]
This is one of my biggest issues with the current code.
However I have no idea how to do this properly then.

I'm open for suggestions.

**Comment by tanteng** [comment]
Illuminate/Foundation/Bootstrap/DetectEnvironment.php line 18：
```
public function bootstrap(Application $app)
{
    if (! $app->configurationIsCached()) {
        $this->checkForSpecificEnvironmentFile($app);

        try {
            (new Dotenv($app->environmentPath(), $app->environmentFile()))->load();
        } catch (InvalidPathException $e) {
            //
        }
    }
}
```

**Comment by marktopper** [comment]
This issue have been fixed in #295
